src/Bubble/index.tsx

```diff
@@ -38,7 +38,7 @@ const Bubble = React.forwardRef<HTMLDivElement, BubbleProps>(({
 
 Bubble.propTypes = {
   /** Specifies contents of the component. */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
we actually have such a case in `Tabs` component, where `Bubble` is rendered without children, caught the error in dev mode